.github/ISSUE_TEMPLATE/Sign.yml

```diff
@@ -0,0 +1,66 @@
+name: Sign Request
+description: File a Sign Request for a plugin
+assignees: codecat
+
+body:
+- type: input
+  id: Name
+  attributes:
+    label: Name
+    description: The Name of your plugin.
+    placeholder: 'ex: JAMP (Just Another Medals Plugin)'
+  validations:
+    required: true
+
+- type: textarea
+  id: Description
+  attributes:
+    label: Description
+    description: The Description of your plugin.
+    placeholder: |
+      Another Medals Plugin, because we didnt have enough of them already!
+    #[![Kappa](https://cdn.frankerfacez.com/emote/480788/1)](https://openplanet.nl/files?search=Medal)
+  validations:
+    required: false
+
+- type: dropdown
+  id: Signed
+  attributes:
+    label: Has This Plugin beed signed before?
+    description: Have you successfuly submited this plugin for signing?
```

Comment:
Typos:
- `submitted`

Probably change `successfully` to `perviously`

Reply:
> Probably change successfully to perviously

I specificaly chose `successfully`, because `previously` doesn't necessarily mean that the plugin got signed

Reply:
Ah well the question asked is if the plug-in was in fact signed before, not if it was merely submitted.

Also there's another typo I missed - "beed" instead of "been"